Client/Frontend/Browser/Tabs/GroupedTabCell.swift

```diff
@@ -268,7 +274,10 @@ class GroupedTabContainerCell: UITableViewCell, UICollectionViewDelegateFlowLayo
 
     @objc func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
         let cellWidth = floor((collectionView.bounds.width - GridTabTrayControllerUX.Margin * CGFloat(numberOfColumns + 1)) / CGFloat(numberOfColumns))
-        let padding = 10
+        let isIpad = UIDevice.current.userInterfaceIdiom == .pad
+        let orientation = UIDevice.current.orientation
+        let isPortrait = (orientation == .portrait || orientation == .portraitUpsideDown)
```

Comment:
There's an easier way to do this, if you want. I added a `.isLandscape` property on UIWindow. :) No need to change it; just letting you know. :)

Reply:
@electricRGB Thanks Roux, didn't know that so I updated it :)